ui/src/app/settings/containers/Repositories/RepositoryForm/RepositoryForm.scss

```diff
@@ -0,0 +1,12 @@
+@import "~vanilla-framework/scss/settings_colors";
+
+.repo-form {
+  margin-top: 0.5rem;
+}
+
+.repo-form__buttons {
+  border-top: 1px solid $color-mid-light;
+  margin-top: 1rem;
+  padding-top: 1rem;
+  text-align: right;
+}
```

Comment:
We should probably turn this into pattern as we have this same code duplicated in a number of places.

Reply:
Good call. I've created [a card for it](https://github.com/canonical-web-and-design/maas-ui/issues/119) that someone can pick up in a follow-up branch.